custom/icds_reports/management/commands/bihar_data_pull.py

```diff
@@ -0,0 +1,124 @@
+from corehq.form_processor.interfaces.dbaccessors import CaseAccessors
+import csv
+
+from django.core.management.base import BaseCommand
+
+from django.db import connections, transaction
+from datetime import date
+from corehq.sql_db.connections import get_icds_ucr_citus_db_alias
+
+
+headers = [
+    'District',
+    'Block',
+    'Sector',
+    'Sector Code',
+    'AWC',
+    'AWC Code (11 Digit)',
+    'Name of Beneficiary',
+    'Age',
+    'Male/Female',
+    'Pregnant or not (If applicable)',
+    'Phone_Number',
+    'adhar number',
+    'Has Bank Account?',
+    'Bank IFSC Code',
+    'Bank Account Number',
+    'Bank Name',
+    'husband Name'
+]
+data_rows = [headers]
+
+
+
+@transaction.atomic
```

Comment:
why do we need this?
Its a select query, should that matter?

Reply:
yep right. my bad